en/docs/security-announcements/security-advisories/2024/WSO2-2023-2864.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Security Advisory WSO2-2023-2864
+category: security-announcements
+published: June 25, 2024
+version: 1.0.0
+---
+
+# SECURITY ADVISORY WSO2-2023-2864
+
+<p class="doc-info">Published: June 25, 2024</p>
+<p class="doc-info">Version: 1.0.0</p>
+---
+
+### AFFECTED PRODUCTS
+* WSO2 API Manager 4.2.0, 4.1.0, 4.0.0, 3.2.0, 3.1.0, 3.0.0, 2.6.0
+* WSO2 Identity Server 6.1.0, 6.0.0, 5.11.0, 5.10.0, 5.9.0, 5.8.0, 5.7.0
+* WSO2 Identity Server as Key Manager 5.10.0, 5.9.0, 5.7.0 
+
+
+### OVERVIEW
+Potential insecure authorization configuration in the OAuth2 flow.
+
+
+### DESCRIPTION
+When the OAuth2 Public Client feature with Client Credentials grant type has been used in a deployment, a malicious actor may obtain the access token without the proper authorization, when the following preconditions are met:
+
+* A service provider must be configured with the `Client Credentials` grant type enabled.
+* The “Allow authentication without the client secret" option must be enabled for the above configured service provider. 
+
+### IMPACT
+If the preconditions mentioned in the description are met, a malicious actor may obtain an access token for the service provider application without providing client secret. There is no impact to your deployment if all the above mentioned preconditions are not met in your configuration.
+
+### SOLUTION
```

Comment:
IMO, we should have two main steps, 1st one is for applying the code level fix which include the public PR for community users and update level info for subscription holders. 2nd one is for applying the recommended configuration.